cloudify_cli/commands/agents.py

```diff
@@ -226,28 +226,35 @@ def get_filters_map(
             for tenant_name in list(tenants_to_deployments):
                 tenant_client = env.get_rest_client(tenant_name=tenant_name)
                 deps_to_execute = tenants_to_deployments[tenant_name]
-
-                node_instances = tenant_client.node_instances.list(
-                    deployment_id=list(deps_to_execute),
-                    _include=['id', 'host_id', 'deployment_id', 'state']
-                )
-
-                # Find all unstarted Compute instances.
-                unstarted_computes = [
-                    ni for ni in node_instances
-                    if ni.id == ni.host_id and
-                    ni.state != _NODE_INSTANCE_STATE_STARTED
-                ]
-
-                for unstarted_ni in unstarted_computes:
-                    logger.info("Node instance '%s' is not in '%s' state; "
-                                "deployment '%s' will be skipped",
-                                unstarted_ni.id, _NODE_INSTANCE_STATE_STARTED,
-                                unstarted_ni.deployment_id)
-                    deps_to_execute.pop(unstarted_ni.deployment_id, None)
-
-                if not deps_to_execute:
-                    del tenants_to_deployments[tenant_name]
+                offset = 0
+                while True:
```

Comment:
i've put the whole thing under this `while` (which is stopped by the `break` later), because this loop is for following pagination